pynest/nest/tests/test_onetooneconnect.py

```diff
@@ -31,11 +31,13 @@
 class OneToOneConnectTestCase(unittest.TestCase):
     """Tests of Connect with OneToOne pattern"""
 
+    def setUp(self):
+        nest.ResetKernel()
+
     def test_ConnectPrePost(self):
         """Connect pre to post"""
 
-        # Connect([pre], [post])
-        nest.ResetKernel()
+        # Connect(pre, post)
```

Comment:
For my taste, these comments can go away, as they just repeat the information already given in the docstring of the function right above. If you agree, please also remove all following occurrences.

Reply:
I agree!